doc/cug.tex

```diff
@@ -2977,15 +2914,42 @@ \subsubsection{Graph Types}
 
 \lstset{language=suiterc}
 \begin{lstlisting}
-[[[ R1/P0Y ]]]    # R[limit]/[interval]
-[[[ R1/P0Y/$ ]]]  # R[limit]/[interval]/[date-time]
-[[[ R1/$ ]]]      # R[limit]/[date-time]
+[[[ R1/P0Y ]]]      # R[limit]/[interval]
+[[[ R1/P0Y/$ ]]]    # R[limit]/[interval]/[date-time]
+[[[ R1/$ ]]]        # R[limit]/[date-time]
 \end{lstlisting}
 \lstset{language=transcript}
 
+\paragraph{Excluding Dates}
+\label{excluding-dates}
+\lstset{language=suiterc}
 
+It can be desirable to exclude a date from a recurrence, this can be achieved
+by appending the date to the recurrence prefixed with an exclamation mark.
+For example \lstinline=[[[ PT1D!20000101 ]]]= means run daily except on the
```

Comment:
Could be more concise, e.g.: "You can use an exclamation mark to exclude a particular date from a recurrence.  For example..."